vueui/src/components/view/records/list.vue

```diff
@@ -4,7 +4,7 @@
     <div v-for="record in filtered" :key="record.id">
       <vui-button :params="{ setactive: record.id }" @click="state.activeview = state.defaultview" push-state>{{ record.id }}: {{ record.name }} ({{ record.rank }})</vui-button>
     </div>
-    <vui-button v-if="(editable & 1) > 0" :params="{ newrecord: 1 }" @click="state.activeview = state.defaultview" push-state>New record</vui-button>
+    <vui-button :params="{ newrecord: 1 }" @click="state.activeview = state.defaultview" push-state>New Record</vui-button>
```

Comment:
Why remove check on client side and show edit button when DM doesn't allow to create record? This v-if checked if console is allowed to create records, what was allowed only to emloyment record console. What actually creates all 3 types of records. Because this new record system is not 3 diffrent systems, but 1 unified one.